pages/guide/08-start.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+group: guide
+title: How to Guide
+---
+
+### How do i get the demo app running on my machine?
+
+Check the [Quick Start Guide](https://yavin.dev/pages/guide/02-start.html)
+
+### How do i connect to a new database?
+
+In path : ```yavin/packages/webservice/app/src/main/resources/demo-configs/db/sql/```
+
+Create an hjson file, example: ```DemoConnection.hjson```
+
+Sample demo of an H2 connection :
+```
+{
+  dbconfigs: [
+    {
+      # We created the connection for the demo using H2 connection
+      name: DemoConnection
+      url: jdbc:h2:mem:DemoDB;  DB_CLOSE_DELAY=-1;  INIT=RUNSCRIPT FROM 'classpath:create-demo-data.sql'
```

Comment:
Only this should be enough for the sample `jdbc:h2:mem:DemoDB;DB_CLOSE_DELAY=-1;`